MovieApp/MovieApp/App/Features/MovieBrowser/MoviesViewController/MovieDetail/MovieDetailViewController.swift

```diff
@@ -33,6 +36,22 @@ class MovieDetailViewController: UIViewController {
         buildViews()
         
         presenter.fetchAll()
+        
```

Comment:
no need for new lines

Reply:
[Removed](https://github.com/lihter/Movie-App/pull/47/files#diff-f5b01144bf2cdd8175add69b29bdfdcf3abf149a38fdd4929f07da30b107f914L39)